spec.emu

```diff
@@ -236,6 +236,15 @@
       <p>An implementation can choose different behaviors for different optional errors.</p>
     </emu-clause>
   </emu-clause>
+
+  <emu-clause id="sec-grammar-notation">
+    <h1>Grammar Notation</h1>
+    <p>This specification follows the same grammar notation convention as defined by <a href="https://tc39.es/ecma262/#sec-grammar-notation">ECMA-262 (Grammar Notation)</a>, with the following caveats:</p>
+    <ul>
+      <li>The terminal symbols of grammars in this specification are individual code points. They are more like ECMA-262 lexical and regexp grammars than the ECMA-262 syntactic grammar.</li>
+      <li>This specification does not use <a href="https://tc39.es/ecma262/#sec-grammatical-parameters">grammatical parameters</a> or <a href="https://tc39.es/ecma262/#sec-lookahead-restrictions">lookahead restrictions</a>, which keeps the grammars simpler.</li>
```

Comment:
```suggestion
      <li>The terminal symbols of grammars defined in this specification are individual code points. This is similar to ECMA-262's <a href="https://tc39.es/ecma262/#sec-lexical-and-regexp-grammars>lexical grammar</a>, rather than to ECMA-262's <a href="https://tc39.es/ecma262/#sec-syntactic-grammar">syntactic grammar</a>.</li>
      <li>This specification does not use <a href="https://tc39.es/ecma262/#sec-grammatical-parameters">grammatical parameters</a> or <a href="https://tc39.es/ecma262/#sec-lookahead-restrictions">lookahead restrictions</a>, to reduce the grammar definitions complexity.</li>
```

Reply:
Much appreciated!